online-workflow/src/main/java/eu/itesla_project/online/StateAnalyzer.java

```diff
@@ -442,7 +423,13 @@ private boolean computePostContingencyState(Network network, String stateId, Con
                 // run load flow on post contingency state
                 logger.info("{}: running load flow on post contingency state {}", stateId, postContingencyStateId);
                 LoadFlowResult result = loadFlow.run();
-                if ( result.isOk() ) {
+                // store post contingency state
+                Integer stateIdInt = Integer.parseInt(stateId);
+                if (parameters.storeStates() || ((parameters.analyseBasecase()) && (stateIdInt == 0))) {
```

Comment:
stateId is, historically and by construction, an integer; Network interfaces deal with states as strings, so some conversions are needed.